es6/react/props.js

```diff
@@ -1,14 +1,13 @@
-import PropTypes from "prop-types";
+import { func, string, shape, objectOf } from "prop-types";
 
-const nosProps = PropTypes.shape({
-  exists: PropTypes.bool.isRequired,
-  getAddress: PropTypes.func.isRequired,
-  getBalance: PropTypes.func.isRequired,
-  claimGas: PropTypes.func.isRequired,
-  send: PropTypes.func.isRequired,
-  testInvoke: PropTypes.func.isRequired,
-  invoke: PropTypes.func.isRequired,
-  getStorage: PropTypes.func.isRequired
+export const nosProps = shape({
+  getAddress: func.isRequired,
+  getBalance: func.isRequired,
+  claimGas: func.isRequired,
+  send: func.isRequired,
+  testInvoke: func.isRequired,
+  invoke: func.isRequired,
+  getStorage: func.isRequired
 });
 
-export default nosProps;
+export const assetProps = objectOf(string);
```

Comment:
Would like to see an isRequired here 👍

Reply:
Putting an isRequired here would be like the equivalent of putting it onto [line 11](https://github.com/nos/api-functions/pull/7/files/072aa9af2d95ca2abb519b8c0ed9122dbab8217b#diff-0189fe5a3008eda6c36c0d9fdeb8881dR11).  We're just defining the shape, it should be up to the implementor whether they want to require it when using it or not.